kvmagent/kvmagent/test/test_ha_pre_fence_vm_on_suspect_host.py

```
@@ -0,0 +1,116 @@
+__ENV_SETUP__ = {
+    'self': {}
+}
+
+
+## describe: case will manage by ztest
+class TestHaPreFenceVmOnSuspectHost(TestCase, pytest_utils.PytestExtension):
+    @classmethod
+    def setUpClass(cls):
+        cls.ha_plugin = HaPlugin()
+        cls.vm_uuid = 'zstac83890-fence-ssh-kill-0001'
+        cls.workdir = '/tmp/zstac83890-fence'
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/zstac83890-fence"

(S108)